fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -198,20 +208,103 @@ private fun NormalTabsTabIcon(normalTabCount: Int) {
+        ) {},
+    )
+    if (shouldShowInactiveButton) {
+        menuItems.add(
+            MenuItem(
+                title = stringResource(R.string.inactive_tabs_menu_item),
+            ) {},
+        )
+    }
+
+    Row(
         modifier = Modifier
             .fillMaxWidth()
-            .height(80.dp)
+            .height(88.dp)
             .background(color = FirefoxTheme.colors.layerAccent),
-        contentAlignment = Alignment.Center,
     ) {
+        IconButton(
+            onClick = {},
+            modifier = Modifier.align(Alignment.CenterVertically),
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_close),
+                contentDescription = stringResource(id = R.string.tab_tray_close_multiselect_content_description),
+                tint = FirefoxTheme.colors.iconOnColor,
+            )
+        }
         Text(
-            text = "Multi selection mode",
-            color = FirefoxTheme.colors.textOnColorPrimary,
+            text = stringResource(R.string.tab_tray_multi_select_title, selectedTabCount),
+            modifier = Modifier.align(Alignment.CenterVertically),
             style = FirefoxTheme.typography.body1,
+            color = FirefoxTheme.colors.textOnColorPrimary,
+            fontSize = 20.sp,
+            fontWeight = FontWeight.W500,
         )
+
+        Spacer(modifier = Modifier.weight(1.0f))
+
+        IconButton(
+            onClick = {},
+            modifier = Modifier.align(Alignment.CenterVertically),
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_tab_collection),
+                contentDescription = stringResource(
+                    id = R.string.tab_tray_collection_button_multiselect_content_description,
+                ),
+                tint = FirefoxTheme.colors.iconOnColor,
+            )
+        }
+
+        IconButton(
+            onClick = {},
+            modifier = Modifier.align(Alignment.CenterVertically),
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_share),
+                contentDescription = stringResource(
+                    id = R.string.tab_tray_multiselect_share_content_description,
+                ),
+                tint = FirefoxTheme.colors.iconOnColor,
+            )
+        }
+
+        IconButton(
+            onClick = { showMenu = true },
+            modifier = Modifier.align(Alignment.CenterVertically),
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_menu),
+                contentDescription = stringResource(id = R.string.tab_tray_multiselect_menu_content_description),
+                tint = FirefoxTheme.colors.iconOnColor,
+            )
+            DropdownMenu(
+                menuItems = menuItems,
+                showMenu = showMenu,
+                onDismissRequest = { showMenu = false },
+            )
```

Comment:
Let's move `DropdownMenu` outside of `IconButton`, so it's near the button but also maintains the appearance behavior of the XML implementation, where it appears over and obscures the buttons. (this is shown in the before screenshot)

Reply:
By moving `DropdownMenu` outside of `IconButton` we'll get something like this: 
![v3nooffset](https://user-images.githubusercontent.com/35462038/233068818-fe3e239e-4060-4520-a3ba-9161238540ae.png)
So we'll have to add another layout containing the menu icon button and keep the `DropdownMenu` at the top, since it will [try](https://developer.android.com/reference/kotlin/androidx/compose/material/package-summary#DropdownMenu(kotlin.Boolean,kotlin.Function0,androidx.compose.ui.Modifier,androidx.compose.ui.unit.DpOffset,androidx.compose.ui.window.PopupProperties,kotlin.Function1)) to expand to the bottom of its parent. We could instead use the `offset` `DropdownMenu` parameter to achieve visual parity:
```suggestion
            DropdownMenu(
                menuItems = menuItems,
                showMenu = showMenu,
                offset = DpOffset(x=0.dp, y=(-24).dp),
                onDismissRequest = { showMenu = false },
            )
```
| V2 menu | V3 offset menu |
| --- | --- |
| ![v2](https://user-images.githubusercontent.com/35462038/233071291-beaeaea0-68ae-464c-bfb4-1f7f0565b9ad.png) | ![v3withoffset](https://user-images.githubusercontent.com/35462038/233071343-a21fdde1-4d38-44c7-8a4e-19f8d60371f8.png) |
What do you think?

Reply:
> since it will try to expand to the bottom of its parent

That's an interesting detail about `DropdownMenu` I wasn't aware of. Yeah, using the offset parameter looks great! Let's go with that for now.